spec/std/enumerable_spec.cr

```diff
@@ -569,7 +569,9 @@ describe "Enumerable" do
   describe "reduce" do
     it { [1, 2, 3].reduce { |memo, i| memo + i }.should eq(6) }
     it { [1, 2, 3].reduce(10) { |memo, i| memo + i }.should eq(16) }
-
+	it { [1, 2, 3].reduce([4]) { |memo, i| [i] + memo }.should eq([4,3,2,1]) }
```

Comment:
These lines need to be indented one more step. Running `crystal tool format` should fix that.

The result is incorrect, it would be `[3, 2, 1, 4]`. The same can be achieved with `unshift` instead of `+` and I'd rather use that because it's more efficient:
```suggestion
	it { [1, 2, 3].reduce([4]) { |memo, i| memo.unshift(i) }.should eq([3, 2, 1, 4]) }
```

Reply:
Ok, i use unshift and simplified it to: `[1, 2, 3].reduce([] of Int32) { |memo, i| memo.unshift(i) }.should eq([3, 2, 1])` (which is the original intention, simple example - reverse the array).